docs/file-reference.md

```diff
@@ -0,0 +1,151 @@
+# OpenCompose file reference documentation
+
+
+OpenCompose file example
+```yml
+version: "0.1-dev"
+
+services:
+   name: foobar
+   containers:
+   - image: foo/bar:tag
+     env:
+     - foo=bar
+     command: ["/bin/foobar"]
+     args: 
+     - "-f"
+     - "/tmp/foobar"
+     ports:
+     - port: 8080:80
+
+```
+
+
+OpenCompose file has two main sections version and services.
+
+## version
+OpenCompose format version
+
+## services
+`services` is main section that lists all the services that this OpenCompose files describes.
+`services` has to be array of [ServiceSpec](#servicespec)
+
+
+### ServiceSpec
+This describes one service. One service doesn't necessarily mean one container.
+One service can be comprised of multiple containers.
```

Comment:
How about: 
Describes one service. Service can be composed of one or multiple containers which are scheduled together and can communicate using localhost. (They share TCP/IP stack.)